components/popover/popover-mixin.js

```diff
@@ -140,6 +145,58 @@ export const PopoverMixin = superclass => class extends superclass {
 				}
 			}
 
+			:host([_mobile][_mobile-tray-location]) .content-width {
+				position: fixed;
+				z-index: 1000;
+			}
+
+			:host([_mobile][_mobile-tray-location="inline-start"]) .content-width,
+			:host([_mobile][_mobile-tray-location="inline-end"]) .content-width {
+				bottom: 0;
+				top: 0;
+			}
+
+			:host([_mobile][_mobile-tray-location="inline-start"]) .content-width {
+				border-bottom-left-radius: 0;
```

Comment:
Currently this is set up to provide the tray position exactly as in `DropdownContentMixin`, which is why these are not using CSS logical properties. The dropdown tray does not support RTL - the consumer specifies left/right/bottom for the tray position, and that's what they get. That may be intentional.

However, I've defined the API for `PopoverMixin` to accept mobile tray location of `inline-start`(left), `inline-block`(right), and `block-end`(bottom) whilst maintaining the existing behaviour. So I am on the fence about going all the way here, by using CSS logical properties which would give full RTL support, vs changing the API to accept left/right/bottom like the existing API, which feels bad but might be intentional.

Reply:
Gotcha. I doubt it was intentional, more likely an oversight. In my mind if someone is positioning it to the left or right, they'd want it flipped in RTL.

Reply:
Ok. I'll take this the next further then and clean up the left/right.